src/Jackett.Common/Services/CacheService.cs

```diff
@@ -19,24 +19,17 @@ public void CacheRssResults(IIndexer indexer, IEnumerable<ReleaseInfo> releases)
         {
             lock (cache)
             {
-                var trackerCache = cache.Where(c => c.TrackerId == indexer.ID).FirstOrDefault();
-                if (trackerCache == null)
-                {
-                    trackerCache = new TrackerCache();
-                    trackerCache.TrackerId = indexer.ID;
-                    trackerCache.TrackerName = indexer.DisplayName;
+                var trackerCache = cache.FirstOrDefault(c => c.TrackerId == indexer.ID) ??
```

Comment:
I think this logic is not the same, you are calling `cache.Add(trackerCache);` in all cases.
The changes in this class are not related to this PR, please undo.

Reply:
Yeah, I see that now. I was attempting to refactor since it was already going to change from the initializer.